aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -419,6 +419,12 @@ struct Vec256 {
   Vec256<T> i0e() const {
     return map(calc_i0e);
   }
+  Vec256<T> i1() const {
+    return map(calc_i1);
```

Comment:
If the vectorized implementations are just mapping the function, is there a reason to implement them at all? We support CPU kernels without vectorized implementations

Reply:
I'll benchmark and remove the psuedo-vectorized implementations if they are not worth it. Thanks!

Reply:
Quick benchmark shows that there is no real gain with `map`. Will remove the vectorized path.

```
[------ i1 : input dtype torch.float32 device cpu ------]
                         |  non-vectorized  |  vectorized
1 threads: ----------------------------------------------
      (128,)             |         4.1      |       4.2  
      (256, 128)         |       679.9      |     706.4  
      (2, 512, 256)      |      5499.7      |    5723.6  
      (2, 64, 256, 128)  |     90534.8      |   89307.4  

Times are in microseconds (us).

[------ i1 : input dtype torch.float64 device cpu ------]
                         |  non-vectorized  |  vectorized
1 threads: ----------------------------------------------
      (128,)             |          7.9     |        8.0 
      (256, 128)         |       1735.0     |     1712.8 
      (2, 512, 256)      |      13505.5     |    13648.6 
      (2, 64, 256, 128)  |     227570.0     |   229265.9 

Times are in microseconds (us).

```

<details>

<summary> Benchmark Code </summary>

```python
import torch
import itertools
import time
from torch.utils.benchmark import Timer
from torch.utils.benchmark import Compare
import sys
import pickle

print('Using pytorch %s' % (torch.__version__))

cuda_shapes = [(128,), (256, 128), (2, 512, 256), (2, 64, 256, 128), (4, 2, 512, 256, 128)]
cpu_shapes = [(128,), (256, 128), (2, 512, 256), (2, 64, 256, 128)]
cpu_dtypes = [torch.float, torch.double] # torch.bfloat16, 
cuda_dtypes = [torch.bfloat16, torch.float16, torch.float, torch.double]
results = []
repeats = 10

for device in ['cpu']:
    if device=='cpu':
        dtypes = cpu_dtypes
        shapes = cpu_shapes
    else:
        dtypes = cuda_dtypes
        shapes = cuda_shapes

    for dtype in dtypes:
        for shape in shapes:
            t = torch.randn(shape, device=device, dtype=dtype)
            
            tasks = [("torch.special.i1(t)", "vectorized", "")]
            timers = [Timer(stmt=stmt, label=f"i1 : input dtype {dtype} device {device}", sub_label=f"{(shape)}", description=desc, globals=globals()) for stmt, desc, label in tasks]

            for i, timer in enumerate(timers * repeats):
                results.append(
                    timer.blocked_autorange()
                )
                print(f"\r{i + 1} / {len(timers) * repeats}", end="")
                sys.stdout.flush()

with open('bessel-compute-vectorized.pkl', 'wb') as f:
    pickle.dump(results, f)

comparison = Compare(results)
comparison.print()

```

</details>

Reply:
I knew you'd actually run the benchmarks ;) Nice follow-up, @kshitij12345

Looks like the difference is in the noise